### PR TITLE
com update tweaks

### PR DESCRIPTION
**Pull request by Avaristimo**
restore idle autoheal
reduce dgun to com to 0
enable combomb prevention widget

### CODE REVIEW

File: luarules/gadgets/mo_preventcombomb.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-local gadgetEnabled = not (Spring.GetModOptions().newdgun or Spring.GetModOptions().unba or Spring.GetModOptions().comupdate)
```
Comment (WatchTheFort) [inline luarules/gadgets/mo_preventcombomb.lua:1]: You actually don't want this gadget enabled. This is responsible for making you lose when you d-gun another Commander.

Comment (Avaristimo) [reply luarules/gadgets/mo_preventcombomb.lua:1]: Isn't that why the "not" is in front?

Comment (WatchTheFort) [reply luarules/gadgets/mo_preventcombomb.lua:1]: It's saying if any of the following modoptions are enabled, disable this gadget: new d-gun, unbacom, comm update. Removing that code means that the gadget will be enabled with the comm update modoption.

Comment (Avaristimo) [reply luarules/gadgets/mo_preventcombomb.lua:1]: Right, meant for it to be there, must have saved the wrong file, thanks for catching this!

**Pull request merged**